django-stubs/views/debug.pyi

```diff
@@ -14,7 +14,11 @@ class CallableSettingWrapper:
     def __init__(self, callable_setting: Union[Callable, Type[Any]]) -> None: ...
 
 def technical_500_response(
-    request: Any, exc_type: Any, exc_value: Any, tb: Any, status_code: int = ...
+    request: HttpRequest,
+    exc_type: Optional[Type[BaseException]],
+    exc_value: Optional[BaseException],
+    tb: Optional[TracebackType],
+    status_code: int = ...,
 ) -> HttpResponse: ...
```

Comment:
The args are all passed on to `ExceptionReporter` which we have types for below

https://github.com/django/django/blob/84206607d6bfd61e7f7a88b51163ffd4153e3b5a/django/views/debug.py#L59-L72